### PR TITLE
Created privacy component

### PR DESCRIPTION
**Pull request by HellraiserCW**

### CODE REVIEW

File: src/app/main/pages/privacy/privacy.component.ts [hunk 1]
```diff
@@ -1,11 +1,72 @@
...
+
+  ngOnInit(): void {
+    this.initPrivacyStatusForm();
+    this.initAccountChangesSubscription();
+  }
+
+  private initPrivacyStatusForm(): void {
+    this.privacyStatusForm = this.fb.group<PrivacyStatusForm>({
+      account: new FormControl(false, { nonNullable: true }),
+      privacyItems: new FormGroup<PrivacyItemsControls>({
```
Comment (GennadiyShpak) [inline src/app/main/pages/privacy/privacy.component.ts:46]: please use FormBuilder

File: src/app/main/pages/privacy/privacy.component.ts [hunk 2]
```diff
@@ -1,11 +1,72 @@
...
+        age: new FormControl(true, { nonNullable: true }),
+        location: new FormControl(true, { nonNullable: true }),
+        description: new FormControl(true, { nonNullable: true })
```
Comment (GennadiyShpak) [inline src/app/main/pages/privacy/privacy.component.ts:49]: please use FormBuilder and it sintaxis

File: src/app/main/pages/privacy/privacy.component.ts [hunk 3]
```diff
@@ -1,11 +1,72 @@
...
+        tap(() => {
+          this.handlePrivacyValueChange(this.account.value);
+        })
+      )
+      .subscribe();
+  }
+
+  private handlePrivacyValueChange(isChecked: boolean): void {
+    if (isChecked) {
+      this.privacyStatusForm.get('privacyItems')?.patchValue(DISABLED_PRIVACY_ITEMS);
```
Comment (GennadiyShpak) [inline src/app/main/pages/privacy/privacy.component.ts:66]: why patchValue?

Comment (HellraiserCW) [reply src/app/main/pages/privacy/privacy.component.ts:66]: fixed

File: src/app/main/pages/privacy/privacy.component.ts (hunk 2 above)
Comment (HellraiserCW) [reply src/app/main/pages/privacy/privacy.component.ts:49]: fixed

File: src/app/main/pages/privacy/privacy.component.ts [hunk 4]
```diff
@@ -1,11 +1,72 @@
...
+
+  private initAccountChangesSubscription(): void {
+    this.account.valueChanges
+      .pipe(
+        tap(() => {
+          this.handlePrivacyValueChange(this.account.value);
+        })
+      )
+      .subscribe();
+  }
```
Comment (GennadiyShpak) [inline src/app/main/pages/privacy/privacy.component.ts:62]: where unsubscribe

File: src/app/main/pages/privacy/privacy.component.html [hunk 5]
```diff
@@ -1 +1,35 @@
...
+<h2 class="privacy__title">Account details</h2>
+<form class="privacy__card" [formGroup]="privacyStatusForm">
+  <div class="privacy__field">
+    <div class="privacy__support-text">
+      <h4 class="privacy__description">Private Account</h4>
+      <p class="privacy__informer">The private account is not viewable to users who are not logged in</p>
+    </div>
+    <label class="privacy__switcher">
+      <input type="checkbox" name="account" [formControl]="account" />
+      <div class="slider round"></div>
```
Comment (GennadiyShpak) [inline src/app/main/pages/privacy/privacy.component.html:14]: empty element

File: src/app/main/pages/privacy/privacy.component.ts (hunk 4 above)
Comment (HellraiserCW) [reply src/app/main/pages/privacy/privacy.component.ts]: fixed

File: src/app/main/pages/privacy/privacy.component.html (hunk 5 above)
Comment (HellraiserCW) [reply src/app/main/pages/privacy/privacy.component.html]: deleted

**Review by GennadiyShpak — APPROVED** [review]

**Pull request merged**